Extension/src/LanguageServer/client.ts

```diff
@@ -3497,7 +3497,8 @@ export class DefaultClient implements Client {
         let workspaceEdits: vscode.WorkspaceEdit = new vscode.WorkspaceEdit();
         let replaceEditRange: vscode.Range | undefined;
         let hasProcessedReplace: boolean = false;
-        const formatUriAndRanges: VsCodeUriAndRange[] = [];
+        const sourceFormatUriAndRanges: VsCodeUriAndRange[] = [];
```

Comment:
Maybe add a comment here explaining that this is not necessarily a "source" file.  It could be a header if the codeAction originated from there.

Reply:
@bobbrow I added a comment.